Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -336,6 +336,17 @@
                                         <Setter Target="PART_RootGrid.Background" Value="Transparent" />
                                     </VisualState.Setters>
                                 </VisualState>
+                                <VisualState x:Name="OverlayVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContent.Visibility" Value="Visible" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="OverlayHidden">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContent" Value="Visible" />
```

Comment:
replace the setters with this. you were missing the Visibility property :)
` <Setter Target="PART_MainContent.Visibility" Value="Visible" />
<Setter Target="PART_ContentOverlay.Visibility" Value="Visible" />`

Reply:
OMG. That makes sense. Thanks, I will fix that.